### ISSUE TITLE
Documentation gives incorrect `client.processor.createProcessorToken` example

### ISSUE THREAD

**Issue by adamcrown** [plaid/plaid-ruby#174]
I'm following the Plaid + Dwolla documentation at https://plaid.com/docs/link/dwolla/ and if you select Ruby as the language, you'll see it gives example Ruby code like.

```ruby
dwolla_response = client.processor.createProcessorToken(access_token, '[Account ID]', 'dwolla')
processor_token = dwolla_response['processor_token']
```

That doesn't seem to work at all, at least not in 4.0.0. I haven't been able to find any way to accomplish getting a Dwolla processor token, with this gem. Stripe seems to be the only supported processor.

Am I missing something?

**Comment by adamcrown** [comment]
I managed to get this working with the following code. Pretty straightforward, thankfully.

```ruby
client.post_with_auth(
  'processor/dwolla/processor_token/create',
  access_token: access_token,
  account_id: account_id
)
```

**Comment by be9** [comment]
Thanks @adamcrown! We will add this to the upcoming new gem version.

**Comment by be9** [comment]
Implemented in #176 (just released in 5.0.0)
